swift/gen-flatbuffers.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+pushd Arrow/Sources/Arrow/
+flatc --swift ../../../../format/Message.fbs
+flatc --swift ../../../../format/Schema.fbs
+flatc --swift ../../../../format/SparseTensor.fbs
+flatc --swift ../../../../format/Tensor.fbs
+flatc --swift ../../../../format/File.fbs
+popd
```

Comment:
Could you prepend our license header to the generated files?

```suggestion
cat <<HEADER > header.swift
// Licensed to the Apache Software Foundation (ASF) under one
// or more contributor license agreements.  See the NOTICE file
// distributed with this work for additional information
// regarding copyright ownership.  The ASF licenses this file
// to you under the Apache License, Version 2.0 (the
// "License"); you may not use this file except in compliance
// with the License.  You may obtain a copy of the License at
//
//   http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing,
// software distributed under the License is distributed on an
// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
// KIND, either express or implied.  See the License for the
// specific language governing permissions and limitations
// under the License.

HEADER
for generated_swift in *_generated.swift; do
  mv ${generated_swift} ${generated_swift}.orig
  cat header.swift ${generated_swift}.orig > ${generated_swift}
  rm ${generated_swift}.orig
done
rm header.swift
popd
```

Reply:
Will do.